app/server/appsmith-server/src/main/resources/application.properties

```
@@ -25,11 +25,13 @@ spring.security.oauth2.client.provider.github.userNameAttribute=login
 # Accounts from specific domains are allowed to login
 oauth2.allowed-domains=${APPSMITH_OAUTH2_ALLOWED_DOMAINS:}
 
-# Segment & Rollbar Properties
-# These properties are intentionally set to random values so that events are not sent to either of them during local development
+# Segment
 segment.writeKey=${APPSMITH_SEGMENT_KEY:}
-com.rollbar.access-token=${APPSMITH_ROLLBAR_ACCESS_TOKEN:}
-roolbar.env = ${APPSMITH_ROLLBAR_ENV:}
+
+# Sentry
+sentry.dsn=${APPSMITH_SENTRY_DSN}
```

```suggestion
sentry.dsn=${APPSMITH_SENTRY_DSN:}
```

The trailing semi-colon is for setting a default value when nothing is present in the environment. In this case the default value will become an empty string. I think this is why all the tests are failing.